src/components/Incidents/AlertsChart/AlertsChart.jsx

```diff
@@ -7,167 +7,187 @@ import {
   ChartGroup,
   ChartLabel,
   ChartThemeColor,
+  ChartTooltip,
+  ChartVoronoiContainer,
 } from '@patternfly/react-charts';
 import { VictoryZoomContainer } from 'victory-zoom-container';
-import { Card, CardTitle } from '@patternfly/react-core';
+import { Bullseye, Card, CardTitle, Spinner } from '@patternfly/react-core';
 import global_danger_color_100 from '@patternfly/react-tokens/dist/esm/global_danger_color_100';
 import global_info_color_100 from '@patternfly/react-tokens/dist/esm/global_info_color_100';
 import global_warning_color_100 from '@patternfly/react-tokens/dist/esm/global_warning_color_100';
+import { createChartBars } from '../utils';
 
-const days = [
-  'Jul 25',
-  'Jul 26',
-  'Jul 27',
-  'Jul 28',
-  'Jul 29',
-  'Jul 30',
-  'Jul 31',
-  'Aug 1',
-  'Aug 2',
-  'Aug 3',
-  'Aug 4',
-  'Aug 5',
-  'Aug 6',
-  'Aug 7',
-  'Today',
-];
-const AlertsChart = () => {
+const AlertsChart = ({ alertsData }) => {
+  const [isLoading, setIsLoading] = React.useState(true);
```

Comment:
Rather than trying to determine what is happening in a higher level component based on the props being passed down, imo we should instead just keep track of the loading state in the `IncidentsPage` component and then pass it down to this component.